src/app/surveys/surveys.component.ts

```diff
@@ -142,7 +142,7 @@ export class SurveysComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   goBack() {
-    this.router.navigate([ '../' ], { relativeTo: this.route });
+    this.router.navigate([ 'manager' ]);
```

Comment:
This is a backwards step because it is better to have relative routes if we change this route in the future.

You should move the `surveys` route from the `home-router.component.ts` to the `manager-router.component.ts` instead and change the link in `manager-dashboard.component.html`

Reply:
Thank you, created a new pull request. [](https://github.com/open-learning-exchange/planet/pull/6581)